guidelines/goals.md

```diff
@@ -1,73 +1,71 @@
 # Goals and Non-Goals
 
-## Goals
-
-The goals define the purpose of the Message Format Working Group (MFWG) and
-inform the decisions about the scope and the priorities of its efforts.
-
-### Data model for defining translations
+This document defines the purpose of the Message Format Working Group (MFWG)
+and informs the decisions about the scope and the priorities of its efforts.
 
-  - Create a description of a data model representing translation units
-    (_messages_). The data model will be agnostic with respect to the file
-    format and the programming environment.
-
-  - Design the data model for representing collections of translation
-    units (_resources_).
+## Goals
 
-  - Make it possible to express complex grammatical features, such as plurals,
-    genders, and inflections, catalogued in the CLDR.
+The primary task of the MFWG is to develop an industry standard for the
+representation of localizable message strings. The design goals are listed
+below.
 
-  - Make it possible to express other variance in translation, due to
-    other linguistic and regional features not listed in the CLDR, as well as
-    due to the presentation media, context, circumstance, and other factors.
+ 1. Express complex grammatical features, such as plurals, genders, and
+    inflections, catalogued in the CLDR.
 
-### Canonical syntax representing the data model
+ 2. Express other variance in translation, due to other linguistic and regional
+    features not listed in the CLDR, as well as due to the presentation media,
+    context, circumstance, and other factors.
 
-  - Design the canonical text representation of the data model, with well
-    defined rules for handling text, special characters, escape sequences,
-    whitespace, and markup.
+ 3. Represent structured data alongside translations, like markup, comments,
+    and metadata.
 
-  - Define parsing errors and the requirements for how to handle them.
+ 4. Ensure interoperability with existing interchange formats, in particular
+    with XLIFF.
```

Comment:
Ah, sorry, I didn't address your comment about SSML. It might be easier to agree on the goals if we don't commit to any formats just yet, and leave that for a future discussion. What do you think?